kubernetes-client/src/main/java/io/micronaut/kubernetes/client/ModelMapper.java

```diff
@@ -38,12 +41,17 @@
  */
 public class ModelMapper {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ModelMapper.class);
+
     // Model's api-group prefix to kubernetes api-group
     private final Map<String, String> preBuiltApiGroups = new HashMap<>();
 
     // Model's api-version midfix to kubernetes api-version
     private final List<String> preBuiltApiVersions = new ArrayList<>();
 
+    // This allows parsing custom (not included in kubernetes core) api versions
+    private final Pattern customVersionParser = Pattern.compile("(V[a-z1-9]+)[A-Z]+[a-zA-Z0-9]+");
```

Comment:
Did you get this regex from somewhere or have you created it? 

I saw that you said this are valid:
- V1MySettings
- V1alpha1Operation

what about: 
- Vtest1Operation

Reply:
I actually created it myself based on all the CRDs I know from my previous experiences but looking slightly more for references, I just found this one: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/version/version.go#L38

Do you think we should use it ?

Reply:
Any update @n0tl3ss ?

Reply:
Hi @cambierr can you update the PR with the regex that you have mentioned.

Reply:
sure, done!